containers/pfcmd/Dockerfile

```
@@ -0,0 +1,18 @@
+ARG KNK_REGISTRY_URL
+ARG IMAGE_TAG
+FROM ${KNK_REGISTRY_URL}/pfdebian:${IMAGE_TAG}
+
+WORKDIR /usr/local/pf/
+
+RUN mkdir -p /usr/local/pf/var/control/
+RUN mkdir -p /usr/local/pf/var/run/
```

Could we merge the two lines?
```suggestion
RUN mkdir -p /usr/local/pf/var/control/ && \
    mkdir -p /usr/local/pf/var/run/
```